frontend/src/app/components/MobileNavigationMenu/mobilenavigation.tsx

```diff
@@ -0,0 +1,34 @@
+'use client'
+
+import React from "react";
+import { menuItemsArr } from "@/app/lib/helpersConstant";
+import './mobilenavigation.scss';
+import Link from "next/link";
+import { handleLogout } from "@/app/lib/utils";
+import { usePathname } from "next/navigation";
+import MobileLogo from "./MobileLogo/MobileLogo";
+
+const MobileNavigation: React.FC = () => {
+    const pathname = usePathname();
+    return (   
+        <>
```

Comment:
Is this something specific for React? Some kind of construction? Just curious..

Reply:
Yep, With ```React.FC``` I tell typescript, that this is gonna be functional component, so component can accept some props, has children, etc.